examples/unresolved-cfd-dem/gas-solid-spouted-bed/gas-solid-spouted-bed.prm

```diff
@@ -114,14 +114,8 @@ end
 #---------------------------------------------------
 
 subsection post-processing
-  set Lagrangian post processing           = false
-  set calculate particles average velocity = false
-  set calculate granular temperature       = false
-  set initial step                         = 100
-  set end step                             = 20000
+  set Lagrangian post-processing           = false
   set output frequency                     = 50
```

Comment:
I don't think this parameter is still relevant in this post-processing section. The output frequency is the same that in the Simulation Control section.

Reply:
Agreed, would be relevant to remove it.